### ISSUE TITLE
Add wxWidgets library support

### ISSUE THREAD

**Issue by antonvw** [appveyor/ci#293]
At this moment anyone who builds an application using wxWidgets and would like to use appveyor for ci, has to provide downloads for the wxWidgets libs from a public url to his project on appveyor, with the complication of mismatches between platform compiled on and appveyor.
It would be great if appveyor provides a precompiled/installed wxWidgets library.

**Comment by FeodorFitsner** [comment]
Does it take too long to download required build from [SF](http://sourceforge.net/projects/wxwindows/files/3.0.2/binaries/)?

**Comment by antonvw** [comment]
No, but it takes time to discover which config is right, at this moment I still get: 'CMake Error at C:/Program Files (x86)/CMake/share/cmake-3.1/Modules/FindPackageHandleStandardArgs.cmake:138 (message):
  Could NOT find wxWidgets (missing: wxWidgets_FOUND)', which I do not get when building locally. And I already spent several hours on other errors.

**Comment by danilaml** [comment]
It certainly would be a very handy as building wxwidgets by itself takes a looong time and downloading custom precompiled libs isn't the msot reliable way to do that.
wxWidgets isn't as big as Qt but a lot of cross-platform projects use it.

**Comment by FeodorFitsner** [comment]
So, what package(s) do we need to add to the server?

**Comment by jieter** [comment]
> Does it take too long to download required build from SF?

No, but it does take long to debug this issue since each build lives in the queue for ~7 min before showing another error message :(

**Comment by FeodorFitsner** [comment]
@jieter What's your account name? Will take a look.

**Comment by jieter** [comment]
Current build: https://ci.appveyor.com/project/jieter/opencpn/build/1.0.34

I'm not comfortable doing these things with windows, and I make stupid typo's all the time. But I have no other way to test this than pushing new commits to GH and wait 15 min for another error to pop up :(...

Is there something like a (vagrant) VM image mimicking the appveyor image?

**Comment by FeodorFitsner** [comment]
You were running on Azure. I've moved account to faster environment - should be better now.

**Comment by jieter** [comment]
@FeodorFitsner thanks! I was guessing you did something to my account, the build started really quick.

**Comment by antonvw** [comment]
As package I would suggest the latest stable wxWidgets version (at this moment 3.0.2), in release and in debug mode

**Comment by jieter** [comment]
It takes some time to build, but this is how I added wxWidgets for OpenCPN build at Appveyor: https://github.com/jieter/OpenCPN/blob/ci/appveyor.yml

ref: OpenCPN/OpenCPN/pull/387
